### PR TITLE
Feat/en 2893 session info add other fields

### PR DESCRIPTION
**Pull request by raduchis**
added txSignPK address to session.info
added -destination-shard-as-observer cli flag

### CODE REVIEW

File: cmd/node/main.go [hunk 1]
```diff
@@ -449,6 +459,18 @@ func startNode(ctx *cli.Context, log *logger.Logger, version string) error {
 		return err
 	}
 
+	output := fmt.Sprintf("%s:%s\n%s:%s\n%s:%s\n%s:%v\n%s:%s\n%s:%s",
```
Comment (sasurobert) [inline cmd/node/main.go]: end it with new line

File: cmd/node/main.go [hunk 2]
```diff
@@ -367,6 +375,19 @@ func startNode(ctx *cli.Context, log *logger.Logger, version string) error {
 	}
 	log.Info("Starting with public key: " + factory.GetPkEncoded(pubKey))
 
+	destinationShardAsObserverString := ctx.GlobalString(destinationShardAsObserver.Name)
+	if destinationShardAsObserverString != metachainShardName {
```
Comment (SebastianMarian) [inline cmd/node/main.go]: You can compare this in insensitive mode using this line:

if !strings.EqualFold(destinationShardAsObserverString, metachainShardName)
or
if strings.ToLower(destinationShardAsObserverString) != strings.ToLower(metachainShardName)

File: cmd/node/main.go (hunk 1 above)
Comment (raduchis) [reply cmd/node/main.go]: fixed

File: cmd/node/main.go (hunk 2 above)
Comment (raduchis) [reply cmd/node/main.go]: fixed

**Pull request merged**